test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildASelfContainedApp.cs

```diff
@@ -94,7 +94,7 @@ public void It_errors_out_when_RuntimeIdentifier_architecture_and_PlatformTarget
         }
 
 		[Fact]
-		public void It_does_not_error_out_when_RuntimeIdentifier_architecture_and_PlatformTarget_do_no_match_but_PlatformTarget_is_AnyCPU()
+		public void It_succeeds_when_RuntimeIdentifier_and_PlatformTarget_mismatch_but_PT_is_AnyCPU()
```

Comment:
You can also pass a "methodName" parameter to the `CopyTestAsset` method if you want a descriptive method name but a shorter folder name.